project/plugins.sbt

```diff
@@ -25,5 +25,7 @@ addSbtPlugin("com.typesafe.sbt" % "sbt-git" % "0.6.2")
 
 addSbtPlugin("pl.project13.scala" % "sbt-jmh" % "0.1.6")
 
+addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.1.7")
```

Comment:
Isn't that one of these plugins better kept globally in your `~/.sbt/0.13/...` files?